pre_processing/set_ecmwf.F90

```diff
@@ -291,150 +279,151 @@ real(dreal) function find_center_time(imager_geolocation, imager_time) &
+
+    type(preproc_nwp_fnames_t), intent(inout) :: nwp_fnames
+    character(len=*), intent(in)              :: cyear, cmonth, cday, chour
+    integer, intent(in)                       :: idx
+    
+    character(len=path_length)                :: ml_dir, sfc_dir
+    
+    print*,cyear, ' ', cmonth, ' ', cday, ' ', chour
+    
+    ml_dir = trim(adjustl(nwp_fnames%nwp_path(idx)))//'an_ml/'// &
+             trim(adjustl(cyear))//'/'// &
+             trim(adjustl(cmonth))//'/'// &
+             trim(adjustl(cday))//'/ecmwf-era5_oper_an_ml_'// &
+             trim(adjustl(cyear))// &
+             trim(adjustl(cmonth))// &
+             trim(adjustl(cday))// &
+             trim(adjustl(chour))//'.'
+             
+    sfc_dir = trim(adjustl(nwp_fnames%nwp_path(idx)))//'an_sfc/'// &
+              trim(adjustl(cyear))//'/'// &
+              trim(adjustl(cmonth))//'/'// &
+              trim(adjustl(cday))//'/ecmwf-era5_oper_an_sfc_'// &
+              trim(adjustl(cyear))// &
+              trim(adjustl(cmonth))// &
+              trim(adjustl(cday))// &
+              trim(adjustl(chour))//'.'
+    
+    ! Set up surface variables   
+    nwp_fnames%ci_f(idx) = trim(adjustl(sfc_dir))//'ci.nc'
+    nwp_fnames%asn_f(idx) = trim(adjustl(sfc_dir))//'asn.nc'
+    nwp_fnames%tcwv_f(idx) = trim(adjustl(sfc_dir))//'tcwv.nc'
+    nwp_fnames%sd_f(idx) = trim(adjustl(sfc_dir))//'sd.nc'
+    nwp_fnames%u10_f(idx) = trim(adjustl(sfc_dir))//'10u.nc'
+    nwp_fnames%v10_f(idx) = trim(adjustl(sfc_dir))//'10v.nc'
+    nwp_fnames%t2_f(idx) = trim(adjustl(sfc_dir))//'2t.nc'
+    nwp_fnames%skt_f(idx) =  trim(adjustl(sfc_dir))//'skt.nc'
+    nwp_fnames%sstk_f(idx) = trim(adjustl(sfc_dir))//'sst.nc'
+    nwp_fnames%cape_f(idx) = trim(adjustl(sfc_dir))//'cape.nc'    
+      
+    ! Set up model level variables
+    nwp_fnames%q_f(idx) = trim(adjustl(ml_dir))//'q.nc'
+    nwp_fnames%t_f(idx) = trim(adjustl(ml_dir))//'t.nc'
+    nwp_fnames%o3_f(idx) = trim(adjustl(ml_dir))//'o3.nc'
+    nwp_fnames%lnsp_f(idx) = trim(adjustl(ml_dir))//'lnsp.nc'
+    nwp_fnames%u_f(idx) = trim(adjustl(ml_dir))//'u.nc'
+    nwp_fnames%v_f(idx) = trim(adjustl(ml_dir))//'v.nc'
+    nwp_fnames%z_f(idx) = trim(adjustl(ml_dir))//'z.nc'
+    
+    print*, trim(nwp_fnames%ci_f(idx))
+    print*, trim(nwp_fnames%asn_f(idx))
+    print*, trim(nwp_fnames%tcwv_f(idx))
+    print*, trim(nwp_fnames%sd_f(idx))
+    print*, trim(nwp_fnames%u10_f(idx))
+    print*, trim(nwp_fnames%v10_f(idx))
+    print*, trim(nwp_fnames%t2_f(idx))
+    print*, trim(nwp_fnames%skt_f(idx))
+    print*, trim(nwp_fnames%sstk_f(idx))
+    print*, trim(nwp_fnames%cape_f(idx))
+      
+    ! Set up model level variables
+    print*, trim(nwp_fnames%q_f(idx))
+    print*, trim(nwp_fnames%t_f(idx))
+    print*, trim(nwp_fnames%o3_f(idx))
+    print*, trim(nwp_fnames%lnsp_f(idx))
+    print*, trim(nwp_fnames%u_f(idx))
+    print*, trim(nwp_fnames%v_f(idx))
+    print*, trim(nwp_fnames%z_f(idx))
+    
+    stop
```

Comment:
Doesn't this terminate the program? This is Fortran, you don't have to explicitly return/stop

Reply:
Yep. Was testing statement left in (outdated) commit.